apps/opencs/view/doc/viewmanager.cpp

```diff
@@ -60,6 +61,40 @@ CSVDoc::ViewManager::ViewManager (CSMDoc::DocumentManager& documentManager)
     mDelegateFactories->add (CSMWorld::ColumnBase::Display_RefRecordType,
         new CSVWorld::IdTypeDelegateFactory());
 
+    // Columns with QLineEdit editor
+    static const CSMWorld::ColumnBase::Display sIdCompletionColumns[] =
+    {
+        CSMWorld::ColumnBase::Display_Cell,
+        CSMWorld::ColumnBase::Display_Class,
+        CSMWorld::ColumnBase::Display_Creature,
+        CSMWorld::ColumnBase::Display_Faction,
+        CSMWorld::ColumnBase::Display_String,
+        CSMWorld::ColumnBase::Display_GlobalVariable,
+        CSMWorld::ColumnBase::Display_Icon,
+        CSMWorld::ColumnBase::Display_Mesh,
+        CSMWorld::ColumnBase::Display_Miscellaneous,
+        CSMWorld::ColumnBase::Display_Music,
+        CSMWorld::ColumnBase::Display_None, // Inplace editing (Table SubView) creates QLineEdit using Display_None
```

Comment:
Because of this: https://github.com/OpenMW/openmw/blob/master/apps/opencs/view/world/util.cpp#L146-L150

Standard Qt view can create editors only using this method. Dialog Subview calls createEditor() method explicitly. Tables make it implicitly through the standard interface.